src/components/originalSampleProcessing/blockProcessing/BlockProcessingLabwarePlan.tsx

```diff
@@ -133,9 +119,11 @@ const BlockProcessingLabwarePlan = React.forwardRef<HTMLDivElement, BlockProcess
       const plannedActions: Source[] | undefined = layoutPlan.plannedActions.get('A1');
       if (plannedActions && plannedActions.length > 0) {
         setFieldValue(`plans.${rowIndex}.sourceBarcode`, plannedActions[0].labware.barcode);
+        setFieldValue(`plans.${rowIndex}.replicateNumber`, plannedActions[0].replicateNumber);
+        setDisableRepNumber(plannedActions[0].replicateNumber!.length > 0);
```

Comment:
As mentioned before, this is not required as we already setting plans.${rowIndex}.replicateNumber with the value .

Reply:
using the  plans.${rowIndex}.replicateNumber check instead will disable the input field once the user enters a value (use case where the source does not have a replicate number, at soon as the user enters a value, the field disables)

Reply:
Ok,that's correct. So disabled status need to be solely based on given layoutPlan I suppose. In that case, please ignore all those comments related.